secretmanager/src/main/java/secretmanager/DeleteSecret.java

```diff
@@ -39,10 +40,22 @@ public void deleteSecret(String projectId, String secretId) throws IOException {
       // Build the secret name.
       SecretName secretName = SecretName.of(projectId, secretId);
 
-      // Create the secret.
+      // Delete the secret.
       client.deleteSecret(secretName);
       System.out.printf("Deleted secret %s\n", secretId);
     }
   }
+
+  // Delete an existing secret.
+  public void deleteSecret(DeleteSecretRequest request) throws IOException {
```

Comment:
This sample should take the name/etags as arguments, and show the user how to construct the resource using it. It should also probably be a new sample in its own file.

Same for the others.

Reply:
Done.